packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -86,21 +86,21 @@ export class LitDevTutorial extends LitElement {
   @state()
   private _preloadedHtml?: {
     idx: number;
-    promise: Promise<{html: string; isOk: boolean}>;
+    promise: Promise<string>;
   };
 
   /**
    * Tutorial manifest json file
    */
   private get _manifest(): TutorialManifest {
-    return this._manifestTask.value?.manifest ?? {steps: [{title: ''}]};
+    return this._manifestTask.value ?? {steps: [{title: ''}]};
   }
 
   /**
    * Message to be displayed if there was an error loading
    */
   private get _manifestLoadError() {
-    return this._manifestTask.value?.isOk === false
+    return this._manifestTask.status === TaskStatus.ERROR
       ? `Could not fetch tutorial manifest. Invalid URL /${this._projectLocation}/`
```

Comment:
Could you move this error directly into the task `error` render function below? The indirection through the getter doesn't seem to be buying much. Same for `_htmlLoadError`.